history/titles/k_barrens.txt

```diff
@@ -270,7 +270,7 @@ c_shehras = {
 }
```

Comment:
Tauren tribes who mostly have tribal holdings shouldn't have clan government now since they don't have any castles.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104963330-ba276100-59f3-11eb-837f-90f705e41790.png)

</details>
The same with Wildhammer dwarves. They should either have castles or tribal governments.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104963422-f1960d80-59f3-11eb-9215-eedffc005da4.png)

</details>
P.S. Try not to force governments in history files. In this case, the game may set the right government by default based on possible governments and your capital holding.